scripts/run-ci-e2e-tests.js

```diff
@@ -288,5 +315,9 @@ try {
     echo(`Killing appium ${APPIUM_PID}`);
     exec(`kill -9 ${APPIUM_PID}`);
   }
+  if (VERDACCIO_PID) {
+    echo(`Killing verdaccio ${VERDACCIO_PID}`);
+    exec(`kill -9 ${VERDACCIO_PID}`);
```

Comment:
Any particular reason for doing `-9` or just copy paste from above?

Reply:
Using `-9` forces the program to be killed and yes, it's reusing the convention from the other `kill` invocations. I'd rather keep it unless there's a good reason not to.